homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -112,7 +121,7 @@ def get(self, request, username):
                 json_response[number] = entity_to_json(
                     self.config, entity, state, brightness)
 
-        return self.json(json_response)
+        return self.json(self.config.response_wrapper(json_response))
```

Comment:
This makes no sense. Why would the sleep app implement a different Hue API than the others ? 

Here is the documentation, and this is what we need to follow: https://developers.meethue.com/develop/hue-api/7-configuration-api/#get-full-state

Reply:
I see it now, you are trying to merge the responses of 2 endpoints in 1 class, please split into a new endpoint.